src/zzz_od/api/routers/drive_disc_dismantle.py

```
@@ -0,0 +1,235 @@
+        "dismantleLevel": config.dismantle_level,
+        "dismantleAbandon": config.dismantle_abandon,
+    }
+
+
+@router.put("/config", response_model=Dict[str, Any], summary="更新驱动盘拆解配置")
+def update_drive_disc_dismantle_config(payload: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    更新驱动盘拆解的配置设置
+
+    ## 功能描述
+    更新驱动盘拆解的配置参数，支持部分更新。
```

Ruff (0.12.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)